src/utils/index.ts

```diff
@@ -102,6 +102,8 @@ import {
   SS58_FORMAT,
 } from '~/utils/constants';
 
+export { cryptoWaitReady } from '@polkadot/util-crypto';
```

Comment:
Why do we need to export this one?

Reply:
They are using it in the dApp when loading keyrings